### PR TITLE
Add Copr nightly test pipeline

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
I am using these as a playground, we do not necessarily need independent jobs in the end. I've expressed this here the options as I see them:

https://github.com/theforeman/forklift/pull/1700#issuecomment-1697466649

**Comment by ehelms** [comment]
This will be adapted based upon the approach settled with the client -- https://github.com/theforeman/jenkins-jobs/pull/351

**Comment by ehelms** [comment]
Proposed order of operations:

 1. Merge this change tat keeps the regular Koji pipeline running but adds the switch to stagingyum
 2. Swap to stagingyum in our CI job (https://github.com/theforeman/jenkins-jobs/pull/365)
 3. Swap to stagingyum in Forklift pipelines (https://github.com/theforeman/forklift/pull/1717)

File: theforeman.org/pipelines/release/pipelines/foreman-rpm.groovy [hunk 1]
```diff
@@ -9,42 +9,114 @@ pipeline {
...
+                            deleteDir()
+                        }
+                    }
+                }
+                stage('staging-install-test') {
+                    agent any
+
+                    steps {
+                        script {
+                            runDuffyPipeline('foreman-rpm', foreman_version)
```
Comment (evgeni) [inline theforeman.org/pipelines/release/pipelines/foreman-rpm.groovy:107]: should/could this pass `staging_source` somehow, so we don't need https://github.com/theforeman/forklift/pull/1717?

**Review by evgeni — APPROVED** [review]

File: theforeman.org/pipelines/release/pipelines/foreman-rpm.groovy (hunk 1 above)
Comment (evgeni) [reply theforeman.org/pipelines/release/pipelines/foreman-rpm.groovy:107]: mhh, `runDuffyPipeline` doesn't allow us to pass arbitrary parameters to the job, so probably not so easy.

Comment (ehelms) [reply theforeman.org/pipelines/release/pipelines/foreman-rpm.groovy:107]: I reached the same conclusion. However, forklift should match CI if someone goes to run things so I think it's OK.

**Pull request merged**